tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -66,15 +67,20 @@ def test_oidc_auth(api_client):
     assert response.status_code == status.HTTP_302_FOUND
 
 
-def test_oidc_logout(api_client):
-    """Test logout url redirects."""
+def test_oidc_logout_without_token(api_client):
+    """Test logging out with token redirects and token is removed."""
```

Comment:
the method name says "logout without token" but the doc string says "logging out with token and token is removed" but the test doesn't seem to mock a token or check for a removed token. Maybe I am missing something?

Reply:
Ok - I think it's just the docstring that is wrong here as I see the logout with token below

Reply:
Ill update the docstring to be less confusing